src/scripts/split-into-songs.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# inspired by https://stackoverflow.com/questions/8544197/splitting-a-file-in-linux-based-on-content
+
+# create a helper to clean artist / song title
+friendly_name () { echo $1 | sed -e 's/\\//g' | sed -e 's/[^a-zA-Z0-9_.-]/-/g' | awk '{print tolower($0)}'; }
+
+# define the path of the file containing all the songs concatenated
+file_path="$SRC_DIR/songs/_all_songs.tex"
+
+# get line numbers for all songs
+line_numbers=$(grep -n -e 'begin{Song}' $file_path | cut -d: -f1)
+
+read -a LINE_NUMBERS<<< $line_numbers
+
+for i in $(seq 0 1 $((${#LINE_NUMBERS[@]}-1))); do
+    start=${LINE_NUMBERS[i]}
+    end=$((${LINE_NUMBERS[i+1]}-1))
+    if [[ $end == -1 ]] # handle last song
+    then
+      end=$(wc -l < $file_path)
+    fi
+    line=$(sed -n "$start{p;q;}" $file_path)
+    line=$(echo $line | sed -e 's/\[\([^]]*\)\]//g')
```

Comment:
Cleaning the song declarations in the form `\begin{Song}[Artist]{The Artist}[Song]{The Song}` to keep only one form. Maybe I should add a comment.

Reply:
What do you mean? Do you have an example of what this did?

Reply:
The way I parse `artist` and `song_title` expects the song definition line to be of the form `\begin{Song}{The Song}{The Artist}` but some of them are of the form `\begin{Song}[Song]{The Song}[Artist]{The Artist}` (e.g. `\begin{Song}[A Maman]{À Maman}[Boulenvrac]{Les Boulenvrac}`)